modules/administration-guide/pages/removing-user-data.adoc

```diff
@@ -21,23 +21,12 @@ Following this procedure makes the service compliant to EU General Data Protecti
 
 .Procedure
 
+WARNING: pass:c,m,a,q[+https:+//__<{prod-host}>__]/swagger/#/user/find_1 and pass:c,m,a,q[+https:+//__<{prod-host}>__]/swagger/#/user/remove endpoints are deprecated. 
+Information about new users is no longer stored in the PostgreSQL database. If the endpoints return `404` this means that there is no user entry in the database and the first two steps of the procedure can be safely skipped.
+
 . Get the __<username>__ user __<id>__ `id`: navigate to pass:c,m,a,q[+https:+//__<{prod-host}>__]/swagger/#/user/find_1, click btn:[Try it out], set *name*: __<username>__, and click btn:[Execute]. Scroll down the *Response body* to find the `id` value.
-// +
-// [subs="+quotes,macros,attributes"]
-// ----
-// $ curl -X GET \
-//   --cookie '_oauth_proxy=__<token>__' \
-//   'pass:c,m,a,q[+https:+//__<{prod-host}>__]/api/user/find?name=__<username>__'
-// ----
 
 . Remove the __<id>__ user data that {prod-short} server manages, such as user preferences: navigate to pass:c,m,a,q[+https:+//__<{prod-host}>__]/swagger/#/user/remove, click btn:[Try it out], set *id*: __<id>__, and click btn:[Execute]. Expect a `204` response code:
-// +
-// [subs="+quotes,macros,attributes"]
-// ----
-// $ curl -i -X DELETE \
-//   --cookie '_oauth_proxy=__<token>__' \
-//   'pass:c,m,a,q[+https:+//__<{prod-host}>__]/api/user/__<id>__'
-// ----
```

Comment:
the curl commands are not supposed to work since API requests are stateful https://github.com/eclipse/che/issues/21419